src/Crowdin.Api/Reports/AddReportSettingsTemplateRequest.cs

```diff
@@ -19,6 +19,9 @@ public abstract class AddReportSettingsTemplateRequest
 
         [JsonProperty("unit")]
         public ReportUnit Unit { get; set; }
+
+        [JsonProperty("isPublic")]
+        public ReportIsPublic IsPublic { get; set; }
```

Comment:
We usually use the built-in `bool` type in such cases. The enum is redundant in this case

Reply:
Using boolean will return true and false in response instead of 0 and 1 as mentioned in [Add Report Settings Template](https://developer.crowdin.com/api/v2/#operation/api.projects.reports.settings-templates.post)
Is there a way to use boolean and get 0 and 1 in request ?

Reply:
@Zahid92 thank you, I didn't notice that, let me check it

Reply:
@Zahid92 checked it with the team, it's going to be changed to a boolean type. The int type was specified by mistake.

Reply:
Thank you @andrii-bodnar I have made the changes.